randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -462,9 +485,11 @@ Extra - asset_id: 12
               Morph Ball
               Morph Ball Bombs or Power Bombs ≥ 3
           All of the following:
-              Speed Booster
+              Speed Booster and Disabled Door Lock Randomizer and Disabled Entrance Randomizer
               Any of the following:
+                  # TODO: Find Video - Short Charge Before Gauntlet Entrance
                   Short Charge (Expert) and Shinespark Damage ≥ 198
+                  # From the Landing Site Pillar: https://youtu.be/0uz1qpr6E2c
                   Shinespark (Intermediate) and Shinespark Damage ≥ 398
```

Comment:
this is a tad difficult, since in order to do this, you need to logically reach the PB room door. Which even without door lock/entrance rando, isnt trivial.
Can we maybe just add this as a todo for now?

Reply:
We could, but if you have shinespark on you can spark up to the ledge and that is well within the bounds for that health.

Reply:
ah thats an option. right. nvm this then